packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -425,19 +434,19 @@ export async function deployPreBuilt({
             bumpedVersion = increment(previousSyncAndActionConfig.version as string | number).toString();
 
             if (runs) {
-                const syncsConfig = await getSyncsByProviderConfigAndSyncName(environment.id, provider_config_key, sync_name);
-                for (const syncConfig of syncsConfig) {
-                    const interval = syncConfig.frequency || runs;
+                const syncs = await getSyncsByProviderConfigAndSyncName(environment.id, provider_config_key, sync_name);
```

Comment:
renaming variable since `getSyncsByProviderConfigAndSyncName` returns syncs not syncConfigs